CONTRIBUTING.md

```diff
@@ -690,6 +690,45 @@ query localConnections {
 curl -s -H "Content-Type:application/json" -H "Authorization: Bearer ${DTX_ACCESS_TOKEN}" http://localhost:26636/gateway/v1/graphql/schema.graphql
 ```
 
+### Invoking the Control Plane Proxy API
+
+The Control Plane Proxy API allows you to make calls to the Confluent Cloud control plane, specifically for artifact and compute pool management endpoints defined by the configuration:
```

Comment:
> specifically for artifact and compute pool management endpoints
Will this section have to be updated every time something is added to the Control Plane Proxy regex pattern? Or is this section just an example focusing on the Flink routes?

Reply:
I think it'd be great to highlight that we can easily add more control plane routes to the proxy by updating the config below.

Reply:
Good question... I would _like_ for that to happen haha but I don't know that we have a way of enforcing that right now.

Reply:
Maybe keep the top-level header and description generic, and use a sub-header for something like "Example: Managing Flink Compute Pools & Artifacts"?

Reply:
oooh good idea, pushing that up